docs_sources/erp_fhirversion_change_20250115-source.adoc

```diff
@@ -0,0 +1,117 @@
+= E-Rezept FHIR Veränderungen für 20250115 image:gematik_logo.png[width=150, float="right"]
+include::./config-source.adoc[]
+
+Ab 15.01.2025 ist das gematik Workflow Package 1.4.0 gültig. Da sowohl DiGA als auch eML Teil des Packages sind, müssen einige Veränderungen vorgenommen werden. Diese Seite bietet eine Zusammenfassung der Veränderungen für die verschiedenen Hersteller.
+
+toc::[]
+
+== Versionsübergabegänge
+Ab dem 01.11.2024 kann der E-Rezept-Fachdienst mehrere aktive Profile haben. Das bedeutet, dass Übergangszeiten mit mehr als zwei aktiven Profilen möglich sind. Unten sind sowohl ein Diagramm als auch eine Tabelle zu sehen, die zeigen, wann welches Profil aktiv ist und bis wann.
+
+image:puml_workflow_profile_version_timeline.png[]
+
+|===
+h|Version  h|Gültig ab  h|Gültig bis
+|Profile 1.2| - | 15.07.2025
+|Profile 1.3| 01.11.2024 | 15.07.2025
+|Profile 1.4| 15.01.2025 | - 
+|Profile 1.5 (Vorläufig)| 15.07.2025 | -
+|===
+
+NOTE: Zwischen dem 15.04.2025 und dem 15.07.2025 sind Profile 1.2 und 1.3 nur für die FdVs verfügbar.
+NOTE: Ab dem 15.07.2025 ist Profil 1.4 nur für die FdVs verfügbar.
+
+== Änderungen
+=== Allgemein
+Betrifft: image:{PVS}[] image:{AVS}[] image:{FdV}[]
+
+* Meta.Profile hochziehen und auf 1.4 ändern. Das betrifft alle Ressourcen.
+* Der PKV- und GKV-Identifier für die verschiedenen Objekte haben jetzt dasselbe System, nämlich: http://fhir.de/sid/gkv/kvid-10
+
+=== Verordnung von DiGAs 
+Betrifft: image:{PVS}[] image:{FdV}[] image:{KTR}[]
+
+* Tasks für Digitale Gesundheitsanwendungen werden mit dem FlowType 162 erstellt.
+* Das CodeSystem link:https://simplifier.net/packages/de.gematik.erezept-workflow.r4/1.4.0-rc1/files/2447794[CS_OrganizationType] hat einen neuen Code, der beim Erstellen einer Task (Task.performerType:Coding) verwendet werden sollte. 
+Nämlich:
+|===
+h|Code h|Display
+|urn:oid:1.2.276.0.76.4.59|Kostenträger
+|===
+
+* Ein neues Profil wird für die Dispensierung eines DiGA-Rezept erstellt: link:https://simplifier.net/packages/de.gematik.erezept-workflow.r4/1.4.0-rc1/files/2447786[GEM_ERP_PR_MedicationDispense_DiGA]. Wichtig zu wissen ist:
+- MedicationDispense.substitution entfällt.
+- MedicationDispense.extension:redeemCode und deepLink kommen hinzu.
+- MedicationDispense.medication ist nicht mehr nur eine Referenz und sieht anders aus.
```

Comment:
Das würde ich genauer beschreiben:
MedicationDispense.medication: Abgabe der Medikationsinformationen erfolgt in einem Identifier und nicht wie bisher als eigenes Medication-Objekt